ogr/ogrsf_frmts/gpkg/ogrgeopackagedatasource.cpp

```diff
@@ -5357,6 +5357,13 @@ int GDALGeoPackageDataset::Create(const char *pszFilename, int nXSize,
                      "table_name TEXT NOT NULL PRIMARY KEY,"
                      "feature_count INTEGER DEFAULT NULL"
                      ")";
+            if (strstr(pszFilename, "_golden") == nullptr)
```

Comment:
what is it this 'golden' hack?

Reply:
for test_ogr_gpkg_write_check_golden_file in autotest/ogr/ogr_gpkg.py

anyway let's close this